src/server/multi_command_squasher.cc

```diff
@@ -138,22 +141,25 @@ bool MultiCommandSquasher::ExecuteSquashed() {
   if (order_.empty())
     return false;
 
+  for (auto& sd : sharded_)
+    sd.replies.reserve(sd.cmds.size());
+
   Transaction* tx = cntx_->transaction;
 
-  if (track_keys_) {
-    tmp_keylist_.assign(collected_keys_.begin(), collected_keys_.end());
-    tx->PrepareSquashedMultiHop(base_cid_, CmdArgList{tmp_keylist_});
```

Comment:
Do you want to delete the overload of `PrepareSquashedMultiHop()` that accepts keys?

Reply:
Done